src/config-import.js

```diff
@@ -475,7 +478,7 @@ function translateMessageObject(warnings) {
             if (
               !validate('translation', {
                 ...value,
-                docRef: { docId: '', versionId: '' },
+                docRef: { docId: 'placeholder', versionId: 'placeholder' },
```

Comment:
don't have enough context for this, but just making sure this is intentional?

Reply:
Yes. Similar to something we do elsewhere in the file:

https://github.com/digidem/comapeo-core/blob/745604ac4ffb33857e7fd5dc623f05057f23729e/src/config-import.js#L193